src/Microsoft.Framework.SecretManager/Program.cs

```diff
@@ -57,9 +58,13 @@ public int Main(string[] args)
             try
             {
                 var app = new CommandLineApplication();
+                app.FullName = "DNX UserSecrets";
                 app.Name = "user-secret";
                 app.Description = "Manages user secrets";
 
+                var assembly = typeof(Program).GetTypeInfo().Assembly;
+                app.ShortVersionGetter = () => assembly.GetName().Version.ToString();
```

Comment:
@MisterJames the full build number shows up only on official builds. For example, grab one of the latest builds from the MyGet feed and you'll see that. On local builds it doesn't know the "build number" so you won't see that.